leaf/extensions/tags.py

```diff
@@ -402,6 +402,55 @@ async def transfer_tag(
                     )
                 )
 
+    @app_commands.describe(tag="The tag you wish to claim.")
+    @app_commands.command(
+        name="claim",
+        description="Claims an unclaimed tag. An unclaimed tag is a tag with no owner "
+        "because they have left the server.",
+    )
+    async def claim_tag(self, interaction: discord.Interaction, tag: str) -> None:
+        async with self.bot.database.transaction():
+            tag_record = await self.bot.database.fetchrow(
+                "SELECT * FROM Tags WHERE name = $1 AND guild_id = $2 AND deleted = FALSE;",
+                tag,
+                interaction.guild.id,
+            )
+
+            if tag_record:
+                try:
+                    member = await interaction.guild.fetch_member(
```

Comment:
Shouldn't we be using `Guild.get_member` instead to get the member from cache?

Reply:
Added a function to bot.py to try members:

```
async def try_member(self, id: int, /, *, guild: discord.Guild) -> discord.Member:
    return guild.get_member(id) or await guild.fetch_member(id)
```